Core/TransactionsImporter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace Core
+{
+	public abstract class TransactionsImporter
+	{
+		public bool CanLoad(Stream file)
+		{
+			try
+			{
+				Load(file);
+			}
+			catch
+			{
+				return false;
+			}
+
+			return true;
+		}
+
+		public abstract List<Transaction> Load(Stream file);
```

Comment:
Codacy found an issue: [Refactor this method to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)